.gitignore

```diff
@@ -0,0 +1 @@
+/test.md
```

Comment:
[Optional]Kindly in .gitignore add `node_modules` and remove the `/` before test.md.